app/commands/github/dispatch_event.rb

```diff
@@ -0,0 +1,11 @@
+class Github::DispatchEvent
```

Comment:
I added this command to simplify the command a bit and to re-use it in the other event dispatching command